src/xversionkeys.dat

```diff
@@ -32,6 +32,10 @@ KEY i BU_XTHIN_VERSION                          0x0002                   0x0003
 KEY i BU_GRAPHENE_FAST_FILTER_PREF              0x0002                   0x0004         u64c
 # Graphene high / low (inclusive) version numbers
 KEY i BU_GRAPHENE_MIN_VERSION_SUPPORTED         0x0002                   0x0005         u64c
+# Mempool sync preferences
+KEY i BU_MEMPOOL_SYNC                           0x0002                   0x0006         u64c
+KEY i BU_MEMPOOL_SYNC_MIN_VERSION_SUPPORTED     0x0002                   0x0007         u64c
+KEY i BU_MEMPOOL_SYNC_MAX_VERSION_SUPPORTED     0x0002                   0x0007         u64c
```

Comment:
@bissias Is having both max and min version supported on 0x0007 a typo? I would think you would want different values for these keys.

Reply:
Good catch! Fixed.